Tests/Functional/Drivers/Queue/AmqpQueueDriverTest.php

```diff
@@ -17,18 +21,39 @@ protected function setUp()
             $this->markTestSkipped('AMQP extension is need for that test');
         }
 
+        if (!static::$kernel) {
+            self::bootKernel();
+        }
+
         parent::setUp();
     }
 
+    public function setMockConsumer($expirationCount)
```

Comment:
AmqpQueueDriver doesn't exist anymore. This test class can be deleted

Reply:
Done.